packages/zod-form-data/README.md

```diff
@@ -190,26 +190,20 @@ const schema = zfd.formData({
 ```
 
 There is a unique case in Remix when using a CustomUploadHandler, 
-the field will be a `File` on the client side, but an ID string (or URL) after uploading on the server.
+the field will be a `File` on the client side, but an ID string / number / url after uploading on the server.
```

Comment:
Maybe this could be
```suggestion
the field will be a `File` on the client side, but an ID string, number, url, etc after uploading on the server.
```